agent/api/statechange_test.go

```diff
@@ -83,3 +84,21 @@ func TestSetTaskTimestamps(t *testing.T) {
 	assert.Equal(t, t2.UTC().String(), change.PullStoppedAt.String())
 	assert.Equal(t, t3.UTC().String(), change.ExecutionStoppedAt.String())
 }
+
+func TestSetContainerRuntimeID(t *testing.T) {
```

Comment:
this is not testing any method.. you should probably test NewContainerStateChangeEvent and NewTaskStateChangeEvent?